### PR TITLE
Add release note for OpenShift SDN upgrade block

### PR DESCRIPTION
**Pull request by jab-rh**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-9972

Link to docs preview: https://73796--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes#ocp-4-16-networking-openshift-sdn-upgrade-block

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jab-rh** [comment]
@zhaozhanqi, this PR adds a release note about blocked cluster upgrades if using OpenShift SDN. What do you think? Thanks!

**Comment by jab-rh** [comment]
@zshi-redhat, this PR adds a release note about blocked cluster upgrades if using OpenShift SDN. What do you think? Thanks!

**Comment by zshi-redhat** [comment]
cc @pperiyasamy

**Comment by zshi-redhat** [comment]
@jab-rh  In 4.16, we will enable limited ovn live migration for external customers. I noticed current suggestion points to the existing offline migration doc, which is fine. we might want to add a pointer to the limited ovn live migration once it is available (the change for live migration can be a separate PR).

looks good to me.

**Comment by jab-rh** [comment]
@zshi-redhat, this is the last that I heard about live migrations: https://issues.redhat.com/browse/OSDOCS-4936

Subsequently, we're now offering this to external customers as well?

Thanks!

**Comment by zshi-redhat** [comment]
> @zshi-redhat, this is the last that I heard about live migrations: https://issues.redhat.com/browse/OSDOCS-4936
> 
> Subsequently, we're now offering this to external customers as well?
> 
> Thanks!

@jab-rh Yes, we will start offering limited ovn live migration to external customers in 4.16: https://issues.redhat.com/browse/OCPSTRAT-1184

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -103,6 +103,24 @@ Warning: short name "ex" could also match lower priority resource examples.test.
 [id="ocp-4-16-networking"]
 === Networking
 
+[id="ocp-4-16-networking-openshift-sdn-upgrade-block"]
+==== OpenShift SDN network plugin blocks future major upgrades
+
+As part of the {product-title} transition to OVN-Kubernetes as the only supported network plugin, if your cluster uses the OpenShift SDN network plugin, starting with {product-title} 4.16, you cannot upgrade to future major versions of {product-title} without migrating to OVN-Kubernetes. For more information on migrating to OVN-Kubernetes, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc#migrate-from-openshift-sdn[Migrating from the OpenShift SDN network plugin].
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:109]:
```suggestion
As part of the {product-title} transition to OVN-Kubernetes as the only supported network plugin, if your cluster uses the OpenShift SDN network plugin, starting with {product-title} 4.16, you cannot upgrade to future major versions of {product-title} without migrating to OVN-Kubernetes. For more information about migrating to OVN-Kubernetes, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc#migrate-from-openshift-sdn[Migrating from the OpenShift SDN network plugin].
```

Comment (mburke5678) [reply release_notes/ocp-4-16-release-notes.adoc:109]: This makes more sense to me, putting the two key phrases together (if you use... you cannot upgrade)
```suggestion
As part of the {product-title} transition to OVN-Kubernetes as the only supported network plugin, starting with {product-title} 4.16, if your cluster uses the OpenShift SDN network plugin, you cannot upgrade to future major versions of {product-title} without migrating to OVN-Kubernetes. For more information about migrating to OVN-Kubernetes, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc#migrate-from-openshift-sdn[Migrating from the OpenShift SDN network plugin].
```

**Comment by mburke5678** [comment]
@jab-rh A couple of comments. Otherwise LGTM.

**Pull request merged**